### PR TITLE
List Of Hotels

### PR DESCRIPTION
**Pull request by RameshPandiS**
#### List off Hotels

### CODE REVIEW

File: src/main/java/in/ramesh/service/HotelDetails.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package in.ramesh.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HotelDetails {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/ramesh/service/HotelDetails.java]: HotelDetailService

File: src/test/java/in/ramesh/test/TestHotelDetails.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import org.junit.Test;
+
+import in.ramesh.service.HotelDetails;
+
+
+public class TestHotelDetails {
+
+	@Test
+	public void testgetHotels(){
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/ramesh/test/TestHotelDetails.java]: testGetHotels - follow naming convention

**Review by ghost — CHANGES_REQUESTED** [review]
Comments Added

**Comment by RameshPandiS** [comment]
errors rectified

**Pull request closed without merging**